test/metabase/api/card_test.clj

```diff
@@ -1453,19 +1453,30 @@
                   :collections       s/Any}
                  (mt/user-http-request :crowberto :get 200 (format "card/%s/related" (u/get-id card)))))))
 
+(defn assert-pivot-card-results-are-correct [result]
```

Comment:
I avoided this because it will cause any assertion failure's message to show in this file, as opposed to where it was called in a test

example, when `(mt/rows result)` fails, the error you get is:

`Query does not have any :rows in results (query_processor_test.clj:316)`

instead of the test it fails in.

Reply:
Good point. Maybe we should change `(mt/rows)` to do a clojure.test `is` assertion rather than throwing an Exception. It seems like that would make it more useful everywhere